refactor(docs-infra): fix template eslint issues in aio examples (#44557)
angular/angular: aio/content/tutorial/toh-pt2.md

### Add a click event binding

Add a `<button>` with a click event binding in the `<li>` like this:

<code-example path="toh-pt2/src/app/heroes/heroes.component.1.html" region="selectedHero-click" header="heroes.component.html (template excerpt)"></code-example>

This is an example of Angular's [event binding](guide/event-binding) syntax.

The parentheses around `click` tell Angular to listen for the `<button>` element's `click` event.
When the user clicks in the `<button>`, Angular executes the `onSelect(hero)` expression.

<div class="callout is-helpful">

> <header>Clickable Elements</header>

Reviewer: ```suggestion
  <header>Clickable elements</header>
```

Author: sorry I was pretty sure I already made this change! :bow: :cry:

### Add a click event binding

Add a `<button>` with a click event binding in the `<li>` like this:

<code-example path="toh-pt2/src/app/heroes/heroes.component.1.html" region="selectedHero-click" header="heroes.component.html (template excerpt)"></code-example>

This is an example of Angular's [event binding](guide/event-binding) syntax.

The parentheses around `click` tell Angular to listen for the `<button>` element's `click` event.
When the user clicks in the `<button>`, Angular executes the `onSelect(hero)` expression.

<div class="callout is-helpful">

> <header>Clickable Elements</header>

Reviewer: ```suggestion
  <header>Clickable elements</header>
```

>   In order to add a click event binding we've added a `<button>` element to our `<li>`, note that this
>   is not the only option, and we could have just as easily added the click event binding directly to the
>   `<li>` element.
> 
>   We can add the click event binding to any possible element, but this is generally not advised as doing
>   so hinders assistive technologies and prevents automatic keyboard behaviors (e.g. the click being emitted
>   also when the user presses the enter key).
> 
>   These problems can still be overcome by adding additional event bindings and ARIA attributes, but by using
>   `<button>` elements we get all the desired behaviors for free.

Reviewer: I suggest keeping this shorter and focused to clicks (since it is a side-note and targets beginners). How about:

```suggestion
  Note that we added the click event binding on a new `<button>` element. While we could have added
  the event binding on the `<li>` element directly, it is better for accessibility purposes to use
  the native `<button>` element to handle clicks.

  For more details on accessibility, see [Accessibility in Angular](guide/accessibility).
```

<header>Clickable Elements</header>

  Note that we added the click event binding on a new `<button>` element. While we could have added
  the event binding on the `<li>` element directly, it is better for accessibility purposes to use
  the native `<button>` element to handle clicks.

>   For more details on accessibility, see [Accessibility in Angular](guide/accessibility).

Reviewer: Nit: I'd leave an empty line above this sentence.

Author: sorry I must removed it by mistake as I copy-pasted your suggestion :sweat:
